test/lib/creation.go

```diff
@@ -107,7 +107,7 @@ func (c *Client) CreateChannelWithDefaultOrFail(channel *messagingv1beta1.Channe
 	err := c.RetryWebhookErrors(func(attempts int) (err error) {
 		_, e := channels.Create(channel)
 		if e != nil {
-			c.T.Fatalf("Failed to create channel %q: %v", channel.Name, e)
+			c.T.Errorf("Failed to create channel %q: %v", channel.Name, e)
```

Comment:
Should we return the error instead of `Errorf`?

/hold

Reply:
And then it will fail after all retries

Reply:
Changed to Logf so that if things don't work after retries, it will fail.